build/org.eclipse.pde.build.tests/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.pde</groupId>
+		<artifactId>eclipse.pde.build</artifactId>
+		<version>4.31.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.pde.build.tests</artifactId>
+	<version>1.4.300-SNAPSHOT</version>
+	<packaging>eclipse-test-plugin</packaging>
+
+	<properties>
+		<!-- This is sued in the ibuilds where the tycho test execution is
+		DISABLED and instead we rely on the test.xml! -->
+		<skipTests>true</skipTests>
+		<ignoredWarnings>${tests.ignoredWarnings}</ignoredWarnings>
+	</properties>
+
+	<profiles>
+		<profile>
+			<id>local-pde-tests</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-maven-plugin</artifactId>
+					</plugin>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>target-platform-configuration</artifactId>
+						<configuration>
+							<dependency-resolution>
+								<extraRequirements>
+									<requirement>
+										<!-- The product -->
+										<type>p2-installable-unit</type>
+										<id>org.eclipse.sdk.ide</id>
+										<versionRange>0.0.0</versionRange>
+									</requirement>
+								</extraRequirements>
+							</dependency-resolution>
+						</configuration>
+					</plugin>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-surefire-plugin</artifactId>
+						<configuration>
+							<product>org.eclipse.sdk.ide</product>
+							<testRuntime>p2Installed</testRuntime>
+							<skipTests>true</skipTests> <!-- this supress the default execution -->
```

Comment:
Isn't this already inherited from the `skipTests` property defined above?

Reply:
properties can be overridden, explicit configurations not :-)
The property is only for the "usual" execution with the ibuild